go/cmd/dolt/commands/log_graph.go

```diff
@@ -372,6 +372,32 @@ func getHeightOfCommit(commit *commitInfoWithChildren) int {
 	return height
 }
 
+func printOneLineGraph(graph [][]string, pager *outputpager.Pager, apr *argparser.ArgParseResults, commits []*commitInfoWithChildren) {
+	decoration := apr.GetValueOrDefault(cli.DecorateFlag, "auto")
+	// print the first commit
+	pager.Writer.Write([]byte(fmt.Sprintf("%s \033[33m commit %s ", strings.Join(graph[commits[0].Row], ""), commits[0].Commit.commitHash)))
```

Comment:
These escape sequences are unfortunate to have spread throughout the code. Can you take a look at leveraging the `color.CyanString()` pattern used elsewhere?

This results in more readable code, and then the NO_COLOR flag will do what users expect. Not sure how difficult this is to change at this point, so feel free to push back :)